### ISSUE TITLE
[Android] Camera no image with AIR 33 for armv8

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#72]
## Problem Description
Camera no image with AIR 33 for armv8 builds.
It has been tested with AIR 26, AIR 28.0.0.125, AIR 29.0.0.122, AIR 31.0.0.103, AIR 32.0.0.89, AIR 32.0.0.109 beta, AIR 33.0.0.182, AIR 33.0.0.212 with captive runtime with multiple Android devices.
With AIR 33.0.0.182 and AIR 33.0.0.212 for armv8 Camera has no image (just black rectangle). For armv7 all works fine for all tested SDKs.

## Steps to Reproduce
Launch AIR application for armv8 architecture with device that has camera. After application launch just grant permission for Camera usage (if it prompted).
```
package {
	import flash.display.Sprite;
	import flash.events.Event;
	import flash.media.Camera;
	import flash.media.Video;
	import flash.events.PermissionEvent;
	import flash.permissions.PermissionStatus;
	
	public class CameraNoImage extends Sprite {
		private var _video:Video;
		private var camera:Camera;
		
		public function CameraNoImage() {
			addEventListener(Event.ADDED_TO_STAGE, init);
		}
		
		private function init(e:Event):void {
			removeEventListener(Event.ADDED_TO_STAGE, init);
			
			camera = Camera.getCamera();
			if (Camera.permissionStatus != PermissionStatus.GRANTED){
				if (camera != null){
					camera.addEventListener(PermissionEvent.PERMISSION_STATUS, cameraPermissionStatusChanged);
					camera.requestPermission();
				}
			} else {
				permissionGranted();
			}
		}
		
		private function cameraPermissionStatusChanged(e:PermissionEvent):void {
			if (e.status == PermissionStatus.GRANTED){
				permissionGranted();
			}
		}
		
		private function permissionGranted():void {
			camera.setMode(640, 480, 30);
			
			_video = new Video(640, 480);
			_video.attachCamera(camera);
			addChild(_video);
		}
	}
}
```

**Actual Result:**
Application show black rectangle instead of camera output.

**Expected Result:**
Application show camera output.

## Known Workarounds
none

**Comment by ajwfrost** [comment]
Problem is caused by a discrepancy in the code where it's looking at supported video formats and finding out what we can convert internally. A format was advertised where the conversion function isn't supported on ARMv8 [yet?] hence removing this format from the compatibility list has meant that we use a different format... works now and will be put into our next release.

Thanks for the detailed error report and sample app to make this easy to reproduce!

**Comment by itlancer** [comment]
Issue fixed with AIR 33.0.1.220.
Thanks.
